utils/scripts/helpers/build_url_repo.sh

```diff
@@ -5,8 +5,9 @@
 set -uex
 
 mydir="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"
-# at some point we want to use: shellcheck source=utils/ci/distro_info.sh
-# shellcheck disable=SC1091
+# at some point we want to use:
+# shellcheck source=utils/scripts/helpers/distro_info.sh
+# shell#check disable=SC1091
```

Comment:
Looks like you've fat-fingered this.
```suggestion
# shellcheck disable=SC1091
```

Reply:
No, that is intentional for that commit.
There will be an updated commit which depends on if checkpatch passes this or fails it.

Reply:
Shellcheck  pragmas should now be ok